content/1.7/back-office/dashboard/dashboard-view.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Dashboard view
+weight: 1
+---
+
+# Dashboard view
+
+Dashboard is the first default page on the back office, in this page the shop performance is displayed.
+
+## Call to action
+
+ - **Demo mode**: this is a toggle switch, it can turn on or off demo mode stats. When in demo mode shows fake stats, when demo mode is turned off, shows the real stats in the dashboard. Has a tooltip: "This mode displays sample data so you can try your dashboard without real numbers."
+
+Below there is a timeline range which can be modified to show the dashboard according to the needed timeline. It can be turned on for:
+
+ - **Day**
+ - **Month**
+ - **Year** - by default it is set to a year.
+ - **Day-1**
+ - **Month-1**
+ - **Year-1**
+
```

Comment:
```suggestion
Choosing an option updates all the blocks of content that are on the Dashboard so that they display data for the chosen period of time only. Blocks that are not stats-based do not change.
```